connector/MongoDBAtlasODBC.pq

```diff
@@ -53,6 +66,29 @@ GetCredentialConnectionString = () as record =>
     in
         CredentialConnectionString;
 
+// Transforms the columns that the driver specifies as SQL_UNKNOWN_TYPE from binary to text values
+shared TransformSqlUnknownColumns = (table) => 
+    let
+        TableType = Value.Type(table),

```

Comment:
I have a feeling that this line is not necessary. 
TableType is not used anywhere in the function
Can you give it a try?

Reply:
Oh thanks for catching that.  I removed the use of `TableType` in my last commit and forgot to also remove it here where it is initialized.